schemas/common_definitions.json

```diff
@@ -110,6 +110,61 @@
             "boolean",
             "integer"
           ]
+        },
+        "date_comparison": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
```

Comment:
I'll need to add a validation check that only id's of a date type can be used here.

Reply:
@dwyeradam Happy with this, but will wait for your validator changes before i approve

Reply:
This will now be done in a separate PR that refactors all validation around routing rules.